### PR TITLE
Upgrade Go to 1.18.4

### PR DESCRIPTION
**Pull request by misberner**

### CODE REVIEW

File: images/stackrox-build.Dockerfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ RUN dnf update -y && \
     wget --quiet -O - https://rpm.nodesource.com/setup_lts.x | bash - && \
     wget --quiet -O - https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
     dnf update -y && \
-    dnf -y groupinstall "Development Tools" && \
+    dnf -y groupinstall "Developmengt Tools" && \
```
Comment (connorgorman) [inline images/stackrox-build.Dockerfile:22]: wrong spelling?

**Review by connorgorman — CHANGES_REQUESTED** [review]
1.18.4 is the highest version downstream

File: images/stackrox-build.Dockerfile (hunk 1 above)
Comment (misberner) [reply images/stackrox-build.Dockerfile]: lol, no idea what happened there. reverted

**Comment by misberner** [comment]
Ah, good catch. Downgraded

**Review by connorgorman — APPROVED** [review]
lg thx

**Review by rukletsov — APPROVED** [review]

**Pull request merged**